ice/http/response.zep

```diff
@@ -32,7 +32,7 @@ class Response implements ResponseInterface
      * @param string body The HTTP response body
      * @param int status The HTTP response status
      */
-    public function __construct(string body = "", int status = 200)
+    public function __construct(string body = null, int status = 200)
```

Comment:
One small thing change type `string` to `var`. 
`string body = null` it's casted to empty string by default

Reply:
Good catch, fixed